lib/android/README.md

```diff
@@ -17,17 +17,58 @@ The following are also required for the compilation of the vendored openssl:
 - Perl; and
 - Clang (to be installed with the NDK, which should be included with CMake when if you use Android Studio).
 
-## Build
+## Build and integrate
+
+Build the `.so` libraries for each supported ABI.
+
+Set environmental variables as needed for Gradle:
+
+```sh
+$ export ANDROID_SDK_ROOT=~/Android/sdk
+```
 
 ```sh
-$ make -C ../ ../target/test/aar.stamp
+$ make -C ../ ../target/test/android.stamp
 ```
 
-### Make variables
+Publish the android library to local maven.
+
+```sh
+$ ./gradlew :didkit:publishToMavenLocal
+```
+
+The AAR file will be built and placed at the following path:
+```
+./didkit/build/outputs/aar/didkit-release.aar
+```
+
+Add mavenLocal() as a repository inside your project's build.gradle
+
+```groovy
+repositories {
+    mavenLocal()
+}
+```
+
+Add didkit library as a dependency inside your app's build.gradle
+```groovy
+dependencies {
+    implementation 'com.spruceid:didkit:0.2.1'
+}
+```
+
+### Example app
+
+Build the example app:
+
+​```sh
+$ ./gradlew :app:build
+​```
 
-- `ANDROID_SDK_ROOT` - path to Android SDK. Default: `~/Android/Sdk`
-- `ANDROID_TOOLS` - Android tools directory. Default is to pick one matching `$(ANDROID_SDK_ROOT)/build-tools/*`.
-- `ANDROID_NDK_HOME` - Android NDK directory. Default is `$(ANDROID_SDK_ROOT)/ndk-bundle)` if exists, or one matching `$(ANDROID_SDK_ROOT)/ndk/*`.
+The APK will be placed in the following location:
+​```
+./app/build/outputs/apk/debug/app-debug.apk
+​```
```

Comment:
There are some zero-width non-breaking spaces in here before the backticks. I added them in the suggestions e.g. in https://github.com/spruceid/didkit/pull/187/files#r684454791, to escape the backticks for the suggestion, as suggested in https://github.com/github/markup/issues/363. But it seems that was not the best idea, because now they are in the file. I don't know how to escape the backticks to suggest change with this interface. git diff shows the characters as `<U+200B>`, which should probably be removed:
```diff
+### Example app
+
+Build the example app:
+
+<U+200B>```sh
+$ ./gradlew :app:build
+<U+200B>```

-- `ANDROID_SDK_ROOT` - path to Android SDK. Default: `~/Android/Sdk`
-- `ANDROID_TOOLS` - Android tools directory. Default is to pick one matching `$(ANDROID_SDK_ROOT)/build-tools/*`.
-- `ANDROID_NDK_HOME` - Android NDK directory. Default is `$(ANDROID_SDK_ROOT)/ndk-bundle)` if exists, or one matching `$(ANDROID_SDK_ROOT)/ndk/*`.
+The APK will be placed in the following location:
+<U+200B>```
+./app/build/outputs/apk/debug/app-debug.apk
+<U+200B>```
```